public/js/index.js

```diff
@@ -1786,6 +1787,14 @@ function initVueComponents(){
     })
 }
 
+function initCtrlEnterSubmit() {
+    $("textarea").keydown(function(e) {
```

Comment:
I don't like this global/general event. What about creating class for that and use it in tmpl where you want to apply this feature?